executor/simple.go

```diff
@@ -235,6 +235,22 @@ func (e *SimpleExec) executeDropUser(s *ast.DropUserStmt) error {
 		if err != nil {
 			failedUsers = append(failedUsers, user.String())
 		}
+
+		// delete privileges from mysql.db
+		sql = fmt.Sprintf(`DELETE FROM %s.%s WHERE Host = "%s" and User = "%s";`, mysql.SystemDB, mysql.DBTable, user.Hostname, user.Username)
```

Comment:
How about use a transaction to run the three delete statements? Or it will leave some grabege when meet something wrong.